### ISSUE TITLE
[Dungeon] - Arcatraz NPC issues

### ISSUE THREAD

**Issue by Paradigmxd** [chromiecraft/chromiecraft#5284]
### What client do you play on?

enGB

### Faction

Alliance

### Content Phase:

Generic

### Current Behaviour

Negaton Screamer removes their School of Magic Shield buff when they cast the School of Magic Volley Bolt (AOE spell)

Spiteful Temptress - When MC the target does not attack other people at all..

Harbringer Skyriss's Split image yields Reputation

### Expected Blizzlike Behaviour

- [x]  ~~Once Negaton Screamer is attack they activate the School of Magic effect based on the type of attack -> e.g. Use Fireball then Fire shield and firebolt volley, etc etc. And should remain with the shield for the duration and NOT REMOVE It when it casts the volley spell~~ 
- [ ]  When MC'ed the target should use spells to attack Players
- [X]  Never seen the Image giving Reputation before?

### Source

https://www.wowhead.com/tbc/npc=20875/negaton-screamer

https://www.wowhead.com/tbc/npc=20883/spiteful-temptress

### Steps to reproduce the problem

Do the dungeon
End

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/5e9b88a9a9ddf510ee2e7a95861988a1fb3e54ca

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Tereneckla** [comment]
You are wrong on the behavior of the negaton screamers [Source](https://youtu.be/JXiFiz9Ga5w?t=1011)
Could you please open a separate issue of the Mind Control behaviour, these 3 issues are not related so they don't belong together
